src/Domain/Entity/Domain.php

```
@@ -222,4 +235,105 @@ public function setNetwork(IPNetwork $network): Domain
+
+                foreach ([ 2 => 'Street', 3 => 'City', 4 => 'State', 5 => 'Postal code'] as $n => $name) {
+                    $this->redacted[] = $this->setRedactedEmptyValue(
+                        "Registrant {$name}",
+                        "$.entities[?(@.roles[{$roleIndex}]=='{$type}'].vcardArray[1][?(@[0]=='adr')][3][{$n}]"
+                    );
+                }
+            }
+        }
+    }
+
+    private function setRedactedEmptyValue(string $type, string $path, ?bool $repacted = false): array
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$repacted'. (undefined)

(UnusedFormalParameter)